<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProjectGlory_V1.0.0.1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>通信</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>691</width>
          <height>59</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btn_conect">
           <property name="text">
            <string>Connect TCP</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_send_tcp">
           <property name="text">
            <string>Send Tcp</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_end_tcp">
           <property name="text">
            <string>End Tcp</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="btn_begin_threadpool">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>410</y>
          <width>187</width>
          <height>57</height>
         </rect>
        </property>
        <property name="text">
         <string>启动线程池</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_init_mpi_noblock">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>280</y>
          <width>187</width>
          <height>57</height>
         </rect>
        </property>
        <property name="text">
         <string>init Mpi no block</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_send_mpi_noblock">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>280</y>
          <width>187</width>
          <height>57</height>
         </rect>
        </property>
        <property name="text">
         <string>send nnoblock msg</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_init_mpi_block">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>200</y>
          <width>187</width>
          <height>57</height>
         </rect>
        </property>
        <property name="text">
         <string>init Mpi</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_mpi">
        <property name="geometry">
         <rect>
          <x>21</x>
          <y>141</y>
          <width>500</width>
          <height>30</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>进程号：</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_send_mpi">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>200</y>
          <width>187</width>
          <height>57</height>
         </rect>
        </property>
        <property name="text">
         <string>send msg</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_end_mpi">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>210</y>
          <width>187</width>
          <height>57</height>
         </rect>
        </property>
        <property name="text">
         <string>End Mpi</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_end_threadpool">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>410</y>
          <width>187</width>
          <height>57</height>
         </rect>
        </property>
        <property name="text">
         <string>关闭线程池</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_send_signal">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>590</y>
          <width>187</width>
          <height>57</height>
         </rect>
        </property>
        <property name="text">
         <string>发送信号</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>数据库</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,1,15">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>15</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lineEdit_str">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>35</height>
             </size>
            </property>
            <property name="text">
             <string>D:\TSP_Database.db</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_opensqlite">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>35</height>
             </size>
            </property>
            <property name="text">
             <string>连接数据</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_encrypt">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="text">
             <string>加密</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_decrypt">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="text">
             <string>解密</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="lineEdit_query">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>35</height>
             </size>
            </property>
            <property name="text">
             <string>select * from secnum</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_query">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>35</height>
             </size>
            </property>
            <property name="text">
             <string>查询</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_sql_model">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>35</height>
             </size>
            </property>
            <property name="text">
             <string>列表显示</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>压缩</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>651</width>
          <height>151</height>
         </rect>
        </property>
        <property name="title">
         <string>压缩目录</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>压缩路径：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_compress">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>35</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_compress">
             <property name="text">
              <string>压缩</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>解压路径：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit__uncompress">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>35</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_uncompress">
             <property name="text">
              <string>解压</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>210</y>
          <width>651</width>
          <height>151</height>
         </rect>
        </property>
        <property name="title">
         <string>压缩数据流</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item row="1" column="0">
          <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
           <item>
            <widget class="QLineEdit" name="lineEdit_compress_data">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="placeholderText">
              <string>文件的全路径（包含名称）</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QPushButton" name="btn_compress_data">
               <property name="text">
                <string>压缩数据流</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btn_uncompress_data">
               <property name="text">
                <string>解压数据流</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>390</y>
          <width>641</width>
          <height>141</height>
         </rect>
        </property>
        <property name="title">
         <string>逐个文件压缩</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item row="0" column="0">
          <widget class="QLineEdit" name="lineEdit_file_list">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="placeholderText">
            <string>文件目录（含有若干文件）</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QPushButton" name="btn_compress_file_list">
             <property name="text">
              <string>压缩</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btn_uncompress_file_list">
             <property name="text">
              <string>解压缩</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QToolButton" name="toolButton">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>550</y>
          <width>111</width>
          <height>101</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="popupMode">
         <enum>QToolButton::DelayedPopup</enum>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="">
       <attribute name="title">
        <string>样式表</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>361</width>
          <height>38</height>
         </rect>
        </property>
        <property name="text">
         <string>测试lineEdit</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>361</width>
          <height>38</height>
         </rect>
        </property>
        <property name="text">
         <string>测试lineEdit(readOnly)</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
